src/pages/create-project/page.jsx

```diff
@@ -1,89 +1,94 @@
-import { useDispatch, useSelector } from 'react-redux';
-import { useNavigate } from 'react-router-dom';
-import ArrowBackIcon from '@mui/icons-material/ArrowBack';
-import { Paper } from '@mui/material';
-import {AddName, AddLabels, AddInfo} from '../../components/add-project/page'
-import { setStep, setFeatures, setProjectName, saveProjectData } from '../../redux/slice/projectSlice';
-import {toast} from 'react-hot-toast'
+import { useDispatch, useSelector } from "react-redux";
+import { useNavigate } from "react-router-dom";
+import ArrowBackIcon from "@mui/icons-material/ArrowBack";
+import { Paper } from "@mui/material";
+import { AddName, AddLabels, AddInfo } from "../../components/add-project/page";
+import {
+  setStep,
+  setFeatures,
+  setProjectName,
+  saveProjectData,
+} from "../../redux/slice/projectSlice";
+import { toast } from "react-hot-toast";
 
 const CreateProject = () => {
   const dispatch = useDispatch();
   const navigate = useNavigate();
-  const { step, projectName, features, companyDetail } = useSelector((state) => state.project);
+  const { step, projectName, features, companyDetail } = useSelector(
+    (state) => state.project
+  );
 
   const handleCheckboxChange = (label, value) => {
     dispatch(setFeatures({ ...features, [label]: value }));
   };
 
-  const handleSave = () => {
-
+  const handleSave = async () => {
```

Comment:
This function is passed as a prop to AddInfoForm to create save the project info in the firebase database. I cant rename it to onSubmit because this function is called in the onSubmit function of the AddInfo form. I think a better aproach would be to handle the project creation in the AddInfoForm component as our state is global, we dont need to pass this as a prop (I addressed these issues in the code before). In the fix I've handled the project creation functionality in the AddInfoForm component.